ci/qunit-selenium-runner.rb

```diff
@@ -0,0 +1,13 @@
+require 'qunit/selenium/test_runner'
+require 'chromedriver/helper'
+
+driver_options = Selenium::WebDriver::Chrome::Options.new
+driver_options.add_argument('--headless')
+driver_options.add_argument('--disable-gpu')
```

Comment:
worth explaining why gpu is disabled

Reply:
https://developers.google.com/web/updates/2017/04/headless-chrome#cli

Reply:
Thanks!

But I meant, I would have added a comment like

`# https://developers.google.com/web/updates/2017/04/headless-chrome#cli indicates this is flag needed for now`